perfkitbenchmarker/vm_util.py

```diff
@@ -95,6 +98,11 @@ class IpAddressSubset(object):
                   'the receiving VM is reachable by internal IP (REACHABLE), '
                   'external IP only (EXTERNAL) or internal IP only (INTERNAL)')
 
+flags.DEFINE_enum('background_network_iptype', IpAddressSubset.EXTERNAL,
```

Comment:
The equivalent for network benchmarks is `--ip_addresses`. Maybe `background_network_ip_address`?